.github/codecov.yml

```diff
@@ -13,20 +13,22 @@
 # limitations under the License.
 
 codecov:
-    strict_yaml_branch: "master"
-    coverage:
-        range: "80...100"
-        status:
-          project:
-            default:
-                target: auto
-                threshold: 0%
-                base: auto
-                only_pulls: true
-          patch:
-            default:
-                target: 100%
-                threshold: 10%
-                base: auto
-                only_pulls: true
-          changes: off
\ No newline at end of file
+  ignore:
+    - "webui/src/app/testing"
+    - "**/*.mock.ts"
+  coverage:
+    range: "80...100"
+    status:
+      project:
+        default:
+          target: auto
+          threshold: 0%
+          base: auto
+          only_pulls: true
+      patch:
+        default:
+          target: 100%
+          threshold: 10%
+          base: auto
+          only_pulls: true
+      changes: off
```

Comment:
does this belong in `.github`? It seems specific to `webui`

Reply:
I haven't figure this out. Content wise, I think it's can be considered not belongs to `webui`, we can just expand the ignore list to cover other files just like how `.gitignore` is. But the main problem is I still haven't be able to get codecov to detect it, and there aren't a config about where this file should be.